packages/fxa-auth-server/test/local/routes/subscriptions.js

```diff
@@ -59,7 +59,6 @@ const stripePlan = require('../payments/fixtures/plan1.json');
 
 let config,
   log,
-  directStripeRoutes,
```

Comment:
I don't want this to be in the same scope as `directStripeRotuesInstance`.  It's now a const where it is used.